src/plugins/explore/public/components/tabs/action_bar/results_action_bar/results_action_bar.scss

```diff
@@ -4,7 +4,7 @@
   padding: $euiSizeXS 0;
 
   &__explore-download-csv-flex-item {
-    margin-left: auto;
+    margin-left: 10px;
```

Comment:
I've changed the React here to be the one aligning items, instead of requiring that "Export" be the thing aligning all the elements to the right.

Reply:
there should be sass variables that we can use

Reply:
Changed to euiSizeS